VAMobile/android/app/src/main/java/gov/va/mobileapp/native_modules/CustomTabsIntentModule.kt

```diff
@@ -15,6 +17,18 @@ import gov.va.mobileapp.R
 class CustomTabsIntentModule(private val context: ReactApplicationContext) :
         ReactContextBaseJavaModule(context) {
 
+    @Throws(java.lang.Exception::class)
+    private fun getCookieManager(): CookieManager? {
```

Comment:
i disagree. since this has to do with auth and login I feel they should be in the same file. Since this is just being called in the authSlice the native module also called in there is RNAuth for iOS and CustomTabs in Android so I feel it is appropriately placed